src/components/Main/Bestseller.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import BestBook from './BestBook';
+
+export class Bestseller extends Component {
+  constructor() {
+    super();
+    this.state = {
+      BestsellerList: [],
+      startIndex: 0,
+      endIndex: 5,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/BestsellerList.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          BestsellerList: data,
+        });
+      });
+  }
+
+  showAfter = () => {
+    const { startIndex, endIndex } = this.state;
+
+    this.setState({
+      startIndex: startIndex + 5,
+      endIndex: endIndex + 5,
+    });
+  };
+
+  showBefore = () => {
+    const { startIndex, endIndex } = this.state;
+    this.setState({
+      startIndex: startIndex - 5,
+      endIndex: endIndex - 5,
+    });
+  };
+
+  render() {
+    const { BestsellerList, startIndex, endIndex } = this.state;
+    const { showBefore, showAfter } = this;
+
+    return (
+      <div className="bestSeller">
+        <h1 className="bestSellerTitle">베스트셀러</h1>
+        <div className="bestSellerGrid">
+          {BestsellerList.slice(startIndex, endIndex).map(BestsellerList => {
+            return (
+              <BestBook
+                key={BestsellerList.id}
+                imgSrc={BestsellerList.imgSrc}
+                rating={BestsellerList.rating}
+                desc={BestsellerList.desc}
+              />
```

Comment:
```suggestion
              <BestBook
                key={BestsellerList.id}
                bestSellerList={BestsellerList}
              />
```
위와 같이 data 자체를 넘겨주고 받는 컴포넌트에서 구조분해해서 사용하시면 편합니다 :)

Reply:
안내 해주신 대로 자식 컴포넌트에서 구조분해할당했습니다! 감사합니다 : )